docs/reference/rest-api/usage.asciidoc

```diff
@@ -285,6 +285,70 @@ GET /_xpack/usage
     "enabled" : true,
     "data_streams" : 0,
     "indices_count" : 0
+  },
+  "data_tiers" : {
+    "available" : true,
+    "enabled" : true,
+    "data_warm" : {
+      "node_count" : 0,
+      "index_count" : 0,
+      "total_shard_count" : 0,
+      "primary_shard_count" : 0,
+      "doc_count" : 0,
+      "total_size_bytes" : 0,
+      "primary_size_bytes" : 0,
+      "primary_shard_size_avg_bytes" : 0,
+      "primary_shard_size_median_bytes" : 0,
+      "primary_shard_size_mad_bytes" : 0
+    },
+    "data" : {
+      "node_count" : 1,
+      "index_count" : 0,
+      "total_shard_count" : 0,
+      "primary_shard_count" : 0,
+      "doc_count" : 0,
+      "total_size_bytes" : 0,
+      "primary_size_bytes" : 0,
+      "primary_shard_size_avg_bytes" : 0,
+      "primary_shard_size_median_bytes" : 0,
+      "primary_shard_size_mad_bytes" : 0
```

Comment:
I don't think we should be reporting the `data` role under `data_tiers`. Trying to do tier routing etc. on the `data` role will fail. I'd rather not have this here as it could be confusing. 

I can see how this could be useful to have, but maybe it should be under another "data" or "data_role" object.

What do you think?

Reply:
Technically it's not a tier, but what I was trying to capture was a way of determining whether the plain "data" role was being used in a deployment (ie, the user left it as the default setting).

I'm not sure what a better format would be for presenting it though, I don't want to stick it in "non_data_tier" for example, because it still contains data and I wouldn't want to count master-only or ingest-only nodes as part of the tier.

I'm open to suggestions though, just not sure of the best way to present it that can still retain the (useful) information

Reply:
What do you think of reporting the usage as `data_roles` instead of `data_tiers`? (there is no `data_role` role, as the non tiered role is called `data` so maybe this won't be confusing)

eg. ```
 "data_roles" : { 
    "data": {
        ...
      },
    "data_hot": {
        ... 
      }
      ...
    }
```